pkg/3scale/amp/operator/highavailability_reconciler.go

```diff
@@ -54,3 +55,48 @@ func HighAvailability(apimanager *appsv1alpha1.APIManager, client client.Client)
 	}
 	return component.NewHighAvailability(opts), nil
 }
+
+func NewBackendExternalRedisReconciler(baseAPIManagerLogicReconciler *BaseAPIManagerLogicReconciler) DependencyReconciler {
```

Comment:
With this reconcilers, `HighAvailabilityReconciler` is not being used, right? can it be removed?